spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -87,7 +94,8 @@ def argv(self):
                 CONF.set('main_interpreter', 'default', True)
                 CONF.set('main_interpreter', 'custom', False)
 
-        is_different_interpreter = pyexec != sys.executable
+        # Part of spyder-ide/spyder#11819
+        is_different_interpreter = is_different_interpreter(pyexec)
```

Comment:
Perhaps the variable on the left shouldn't have the same name as the new function to avoid the odd errors generated in Travis?

So let's change this to

    is_different = is_different_interpreter(pyexec)

And also change the usage of `is_different_interpreter` below.